tests/bluetooth/bsim/mesh/src/test_blob.c

```diff
@@ -27,6 +27,22 @@ static enum {
 	XFER_GET_FAIL = 1
 } msg_fail_type;
 
+static void test_cli_init(void)
+{
+	/* Ensure those test devices will not drift more than
+	 * 100ms for each other in emulated time
+	 */
+	tm_set_phy_max_resync_offset(100000);
```

Comment:
With this change other `tm_set_phy_max_resync_offset` calls in this file are not needed and can be removed.